spec/tbd_tests_spec.rb

```diff
@@ -1248,7 +1275,7 @@
       if id == "Open area 1 Floor"
         expect(surface[:kiva]).to eq(:basement)
         expect(surface).to have_key(:exposed)
-        expect(surface[:exposed]).to be_within(TOL).of(8.70) # 12.6 - 3.9
+        expect(surface[:exposed]).to be_within(TOL).of(12.59) # not 8.70
```

Comment:
Exposed perimeter correctly set. Initial EnergyPlus runs (this model, plus others) without warnings or errors.